run.py

```diff
@@ -87,12 +99,55 @@ def check_pip_deps():
 
 
 def config_validator():
-    '''Make sure the config file is ready.'''
+    """Make sure the config file is ready."""
     # Checking if oauth token is configured
     if app.config['OAUTH_TOKEN'] == '':
         sys.exit(TOKEN_HELP)
 
 
+def initialize_status(statuses):
+    """Map the status name with the milestone id on GitHub.
+
+    The project needs the mapping in between milestones and their id
+    to be able to query status related things. It uses a backup version
+    when the request to GitHub fails.
+    """
+    print('Statuses Initialization…')
+    REPO_ROOT = app.config['ISSUES_REPO_URI'].rpartition('/issues')[0]
+    milestones_path = os.path.join('repos', REPO_ROOT, 'milestones')
+    milestones_url = urlparse.urlunparse(
+        ('https', 'api.github.com', milestones_path, '', '', ''))
+    milestones_file = os.path.join(app.config['DATA_PATH'], 'milestones.json')
+    try:
+        # Get the milestone from the network
+        print('Fetching milestones from Github…')
+        r = requests.get(milestones_url)
+        milestones_content = r.content
+        if r.status_code == 200:
+            with open(milestones_file, 'w') as f:
+                f.write(r.content)
+            print('Milestones saved in data/')
+        r.raise_for_status()
+    except requests.exceptions.HTTPError as error:
+        # Not working, let's use the cached copy
+        print(MILESTONE_ERROR.format(msg=error))
+        with open(milestones_file, 'r') as f:
+            milestones_content = f.read()
+    finally:
+        # save in data/ the current version
+        print('Milestones in memory')
+        app.config.update(STATUSES=convert_milestones(milestones_content))
+        app.config.update(JSON_STATUSES=json.dumps(app.config['STATUSES']))
```

Comment:
If we don't want to create an additional config object that's JSON encoded, the other option is to dump it it as a template variable here:

https://github.com/webcompat/webcompat.com/blob/master/webcompat/views.py#L235

But, it might be useful to have this in JSON elsewhere, so